src/controller/python/chip/crypto/p256keypair.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <inttypes.h>
+
+#include <crypto/CHIPCryptoPAL.h>
+#include <native/PyChipError.h>
+
+typedef bool (*pychip_ECDSA_sign_msg)(void * pyObject, const uint8_t * msg, size_t msg_length, uint8_t * out_signature,
+                                      uint32_t * signature_length);
+
+typedef bool (*pychip_ECDH_derive_secret)(void * pyObject, const uint8_t * remote_public_key, uint8_t * out_secret,
+                                          uint32_t * out_secret_length);
+
+namespace chip {
+namespace python {
+
+class pychip_P256Keypair : public Crypto::P256Keypair
```

Comment:
Missing override for `Pubkey()` (see ECKeypair)

Reply:
done